<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>ChangePasswordDialog</class>
 <widget class="QDialog" name="ChangePasswordDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Change Administrator Password</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>11</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="3" column="2" >
    <widget class="QPushButton" name="cancelButton" >
     <property name="text" >
      <string>Cancel</string>
     </property>
     <property name="shortcut" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="textLabel3" >
     <property name="text" >
      <string>Confir&amp;m New Password</string>
     </property>
     <property name="buddy" >
      <cstring>confirmNewLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="textLabel1" >
     <property name="text" >
      <string>Old Password</string>
     </property>
     <property name="buddy" >
      <cstring>oldLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="textLabel2" >
     <property name="text" >
      <string>New Password</string>
     </property>
     <property name="buddy" >
      <cstring>newLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2" >
    <widget class="QLineEdit" name="oldLineEdit" >
     <property name="echoMode" >
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2" >
    <widget class="QLineEdit" name="newLineEdit" >
     <property name="echoMode" >
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2" >
    <widget class="QLineEdit" name="confirmNewLineEdit" >
     <property name="echoMode" >
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QPushButton" name="okButton" >
     <property name="text" >
      <string>&amp;OK</string>
     </property>
     <property name="shortcut" >
      <string>Alt+O</string>
     </property>
     <property name="default" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>oldLineEdit</tabstop>
  <tabstop>newLineEdit</tabstop>
  <tabstop>confirmNewLineEdit</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>ChangePasswordDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>275</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel" >
     <x>93</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ChangePasswordDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>439</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel" >
     <x>431</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
